http2/src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2Connection.java

```diff
@@ -137,7 +140,11 @@ private HttpFields buildHeaders(URL url, HeaderManager headerManager, CookieMana
                     String n = header.getName();
                     // Don't allow override of Content-Length
                     // TODO - what other headers are not allowed?
-                    if (!HTTPConstants.HEADER_CONTENT_LENGTH.equalsIgnoreCase(n)) {
+                    if(n.startsWith(":")){
+                        LOG.warn("The specified pseudo header "+ n +" may not be supported by the"

```

Comment:
Use slf4j syntax for log concatenation which is more performant (when WARN is not enabled for example in logs):
```
LOG.warn("The specified pseudo header {} is not allowed and will be ignored", n);
```